Gems/EMotionFX/Code/EMotionFX/Tools/EMotionStudio/Plugins/StandardPlugins/Source/AnimGraph/AnimGraphModel.cpp

```diff
@@ -1105,6 +1105,10 @@ namespace EMStudio
         }
 
         delete modelItemData;
+        if (m_modelItemDataSet.empty())
+        {
+            emit RemoveAll();
```

Comment:
I don't think we need a custom emit signal like this. During the remove of modelItemData it is already emit remove signal.